### ISSUE TITLE
Split cjdcmd in to individual utilities

### ISSUE THREAD

**Issue by inhies** [inhies/cjdcmd#23]
Cjdcmd is big and getting bigger. Some users have requested that cjdcmd be split in to single function utilities, like the traditional Linux ping and traceroute programs. 

This is possible, however naming becomes an issue. We can't just name these utilities 'ping' and such because those already exist. Does anyone have some ideas for a pre or post-fix that can be added to the functions to distinguish them as belonging to cjdcmd? 

Keep in mind the proposed name change to core or cor3 as in issue #2

**Comment by ryansb** [comment]
I think more important than splitting it into utilities would be logically splitting it into subcommands the way `git` or `hg` do it.

**Comment by thefinn93** [comment]
Yeah, the way git and hg do it is great. Splitting it off into shit would just get confusing.
